Dockerfile

```diff
@@ -1,6 +1,6 @@
 # To build run: docker build -t avro-schema-registry .
 
-FROM ruby:2.4.2
+FROM ruby:2.6.1
```

Comment:
If there is no ruby:2.6.2 image yet, you might want to add a `RUN gem update --system` line to address the Rubygems vulnerabilities: https://blog.rubygems.org/2019/03/05/security-advisories-2019-03.html

Reply:
According to https://github.com/docker-library/ruby/issues/273 it was already handled in a newer version of the `ruby:2.6.1` image.

Reply:
Looks like the 2.6.2 Docker image has been published so we can just go to that :)